SDLTMRepair/SDLTMRepair/TellMe/TellMeProvider.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Sdl.TellMe.ProviderApi;
+
+namespace Sdl.Community.TMRepair.TellMe
+{
+	[TellMeProvider]
+	public class TellMeProvider: ITellMeProvider
+	{
+		public string Name => string.Format(PluginResources.TellMe_Provider, PluginResources.Plugin_Name);
+
+		public AbstractTellMeAction[] ProviderActions => new AbstractTellMeAction[]
+		{
+			new CommunityWikiAction
+			{
+				Keywords = new[] { "repaire", "sdltm repair", "support", "wiki" }
```

Comment:
these are word keys, therefore should be:
"sdltm", "repair"
same for the others